Volume/Views/Onboarding/OnboardingView.swift

```diff
@@ -87,6 +88,7 @@ struct OnboardingView: View {
                     Button("Start reading") {
                         withAnimation(.spring()) {
                             isFirstLaunch = false
+                            AppDevAnalytics.logEvent(CompleteOnboarding())
```

Comment:
Take this out of the `withAnimation` block (I'd put it right before on line 89). Same thing with the event on 121 below.

Reply:
thx I have no idea how swift UI codeblocks work 😅